packages/helix_editor.rb

```diff
@@ -0,0 +1,73 @@
+require 'package'
+
+class Helix_editor < Package                 # The first character of the class name must be upper case
+  description 'A vim like editor with LSP support'
+  homepage 'https://helix-editor.com/'
+  version '22.12'
+  license 'MPL-2.0' # license of source
+  compatibility 'all'
+  source_url 'https://github.com/helix-editor/helix/archive/refs/tags/22.12.tar.gz'
+  source_sha256 'edae8af46401b45c3e71c38b4fa99f931c4458127978ccd1b29aaae79331d972'   # Use the command "sha256sum"
+  #source_url 'file:///usr/local/home/helix/deleteme.tar.gz'
+  #source_sha256 'c3a0e022a7bcd6b139ae7abf1e0dae33c8f9ec82dc680076d9e5ea6593b85bc6'
+
+  depends_on 'rust' => :build
+
+  # TODO is it the appropriate location?
+  # By default, helix expects runtime to be in ~/.config or XDG_CONFIG_HOME.
+  # The first location is not appropriate because helix will not be installed in ~/
+  # With the second location crew complaints that it is not  FHS3 compliant.
+  @runtime_dest_dir_suffix = "/var/lib/helix"
```

Comment:
Everything ChromeOS touches needs to be under `/usr/local`...

Reply:
Yes and this extra directory is not even necessary if the output of the build is a single executable.

Reply:
Yes all the files normally go under `/usr/local/`.

Maybe the code comments are a bit unclear, but `@runtime_dest_dir_suffix` is not used as a full path. It is used as a suffix, and appended after a prefix as in: `"#{CREW_DEST_PREFIX}#{@runtime_dest_dir_suffix}"`.

Also, *helix editor* is in two parts:
- The `hx` executable, that would go inside `/usr/local/bin` I suppose.
- The `runtime` directory that is needed by `hx` in order to work properly. Without it, the themes will not be available, and the code syntax highlighting will not work either.

By default, helix expects the runtime folder to be found either under `$XDG_CONFIG_HOME/helix` (which corresponds to `/usr/local/.config/helix`), either under `~/.config/helix`, or both.
`~/.config/helix` is outside of `/usr/local` so I should not touch it. Crew is not happy with `/usr/local/.config/helix` because it is not "FHS3 compliant".
This is why I am unsure of where to put this runtime folder. Currently I use `/usr/local/var/lib/helix`.
To allow helix to find its runtime folder I set the environment variable HELIX_RUNTIME.

Reply:
There is a FHS override flag you can put in the package file.

Reply:
Ok. I use the `no_fhs` flag now. The script is also simpler.